perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -54,6 +76,93 @@
                   'num_machines': 1}
 
 
+JOB_FILE_TEMPLATE = """
+[global]
+ioengine=libaio
+invalidate=1
+blocksize=4k
+direct=1
+runtime=10m
+time_based
+filename={{filename}}
+do_verify=0
+verify_fatal=0
+randrepeat=0
+size={{size}}
+
+{% for rwkind in ('randread', 'randwrite') %}
+{% for iodepth in iodepths %}
+[{{rwkind}}-io-depth-{{iodepth}}]
+stonewall
+rw={{rwkind}}
+iodepth={{iodepth}}
+
+{% endfor %}
+{% endfor %}
+"""
+
+
+def GetIODepths(io_depths):
+  """Parse the io_depths parameter.
+
+  Args:
+    io_depths: a string in the format of the --io_depths flag.
+
+  Returns:
+    An iterable of integers.
+
+  Raises:
+    ValueError if FLAGS.io_depths doesn't follow a format it recognizes.
+  """
+
+  match = IODEPTHS_REGEXP.match(io_depths)
+
+  if match.group(2) is None:
+    return [int(match.group(1))]
+  else:
+    return range(int(match.group(1)), int(match.group(3)) + 1)
+
+
+def WriteJobFile(mount_point):
+  """Write a fio job file.
+
+  Args:
+    mount_point: the mount point of the disk we're testing against.
+
+  Returns:
+    The contents of a fio job file, as a string.
+  """
+
+  if FLAGS.against_device:
+    filename = mount_point
+    size = FLAGS.device_fill_size
+  else:
+    filename = posixpath.join(mount_point, DEFAULT_TEMP_FILE_NAME)
+    size = '100G'
+
+  return jinja2.Template(JOB_FILE_TEMPLATE).render(
```

Comment:
Nit: no need to change, but [`VirtualMachine.RenderTemplate(...)`](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/virtual_machine.py#L334) is similar.

If you leave as-is, could you set `undefined=jinja2.StrictUndefined` in the Template constructor? Otherwise missing variables will be left blank, which can be hard to debug.